src/components/contentBlocks/quotesSlide.tsx

```diff
@@ -0,0 +1,90 @@
+import { graphql } from 'gatsby';
+import Image, { FluidObject } from 'gatsby-image';
+import React, { FC } from 'react';
+import styled, { css } from 'styled-components';
+
+import { QuoteInformationFragment } from '../../../types/graphql-types';
+import { applyMediaQueryMd, ScreenSize } from '../../style/dimensions';
+import ContentfulRichText from '../contentfulRichText';
+import { CarouselSlide } from '../elements/carousel';
+import Flex from '../elements/flex';
+import Paragraph from '../elements/paragraph';
+
+interface Props {
+  data?: QuoteInformationFragment | null;
+}
+
+export const slideWidth: Record<ScreenSize, number> = {
+  sm: 310,
+  md: 530,
+  lg: 530,
+};
+
+export const slideMarginRight: Record<ScreenSize, number> = {
+  sm: 10,
+  md: 24,
+  lg: 24,
+};
+
+const Slide = styled(CarouselSlide)`
+  height: fit-content;
+`;
+
+const SlideInner = styled.div`
+  margin: 20px;
+
+  ${applyMediaQueryMd(css`
+    margin: 30px;
+  `)}
+`;
+
+const AuthorImage = styled(Image)`
+  height: 64px;
+  margin-right: 13px;
+  min-width: 64px;
+  width: 64px;
```

Comment:
As mentioned earlier a `border-radius: 50% 50%;` could be helpful to auto-crop the images into circles.